pkg/sql/sem/tree/name_resolution.go

```diff
@@ -576,6 +576,12 @@ type CommonLookupFlags struct {
 	AvoidCached bool
 }
 
+// SchemaLookupFlags is the flag struct suitable for GetSchema().
+type SchemaLookupFlags struct {
+	CommonLookupFlags
```

Comment:
I think its time to just add requiremutable to the common lookupflags

Reply:
I was hoping to avoid updating all the existing usages of `RequireMutable` on `tree.ObjectLookupFlags` for now, but I'm willing to reconsider. If I don't do it now, I'll do it this week.